src/Data/Textile/Db.elm

```diff
@@ -53,5 +42,18 @@ decode =
                                 (Decode.field "materials" (Material.decodeList processes))
                                 (Decode.field "products" (Product.decodeList processes))
                                 (Decode.field "transports" Transport.decodeDistances)
+                                |> Decode.andThen
+                                    (\partiallyLoaded ->
+                                        let
+                                            result =
+                                                Process.loadWellKnown processes
+                                        in
+                                        case result of
+                                            Ok wellKnown ->
+                                                Decode.succeed (partiallyLoaded wellKnown)
+
+                                            Err err ->
+                                                Decode.fail err
+                                    )
```

Comment:
The same way you've used `RemoteData.fromResult`, you could use `Json.Decode.Extra.fromResult`:

```elm
import Json.Decode.Extra as DE

    |> Decode.andThen
        (\partiallyLoaded ->
            processes
                |> Process.loadWellKnown
                |> Result.map partiallyLoaded
                |> DE.fromResult
        )
```

Reply:
ah yes true I wanted to do that, and forgot, thanks for the spot!